include_core/j9nongenerated.h

```diff
@@ -199,6 +199,10 @@ typedef struct J9MemorySegment {
 #define MEMORY_TYPE_DEFAULT                 0x2000000
 #define MEMORY_TYPE_FROM_JXE                0x4000000
 #define MEMORY_TYPE_SHARED_META             0x8000000
+/* On Linux, memory marked as MEMORY_TYPE_DISCLAIMABLE_TO_FILE can be disclaimed
+ * to a backing file with madvise().
+ */
+#define MEMORY_TYPE_DISCLAIMABLE_TO_FILE    0x10000000
```

Comment:
I don't see any uses of `MEMORY_TYPE_DISCLAIMABLE_TO_FILE`; what is the point of this definition?

Reply:
OpenJ9 will use that.